lms/templates/bulk_email/unsubscribe_success.html

```diff
@@ -34,8 +35,9 @@ <h1>
 
             %endif
             <br><br>
-            ${Text(_("{link_start}Return to edX.org{link_end}")).format(
-            link_start=HTML("<u> <a href='https://www.edx.org/'>"),
+            ${Text(_("{link_start}Return to {platform_name}{link_end}")).format(
```

Comment:
I am speaking with @cc-okoro about the impact of changing this, the sole usage of `edX.org` hardcoded where it shouldn't be, to using the already-abstracted `platform_name`

Reply:
I did some sleuthing for this page on edx.org and found that it's an unused page as edX has switched to Braze.